orchestra/todos/forms.py

```diff
@@ -0,0 +1,6 @@
+from django import forms
+
+
+class ImportTodoListTemplateFromSpreadsheetForm(forms.Form):
```

Comment:
You could imagine having this form do the validation on the URL here (it should look like a google spreadsheets URL), so you don't have to do it in your `get_csv_from_spreadsheet` func.

Reply:
I've cleaned up/removed the assertion logic in `get_csv_from_spreadsheet` and recast it as a validation error that gets reported in the form. So not quite what you suggest, but same effect! :)